qlib/log.py

```diff
@@ -12,7 +12,45 @@
 from .config import C
 
 
-def get_module_logger(module_name, level: Optional[int] = None):
+class MetaLogger(type):
+    def __init__(self, name, bases, dic):
+        super().__init__(name, bases, dic)
+
+    def __new__(cls, name, bases, dict):
+        wrapper_dict = type(logging.getLogger("MetaLogger")).__dict__.copy()
+        wrapper_dict.update(dict)
+        wrapper_dict["__doc__"] = logging.getLogger("MetaLogger").__doc__
```

Comment:
Why not use `logging.Logger.__doc__` ?

Reply:
Yes we can use `logging.Logger.__doc__` here, which will have the same results as doing so with an instance.

Reply:
Why should we create a Logger instance  here instead of using the class? 
Using the class directly takes fewer steps and the code looks simpler